### DISCUSSION TITLE
missing style.css in v9.0.4

### DISCUSSION THREAD

**Discussion by jsmith8954** [gpbl/react-day-picker#2328] [Support] [answered]
Upgrading to `9.0.4` from version 8, and following the upgrade guide.

it mentions to swap to the styling sheet to import to our typescript file

```
import "react-day-picker/style.css";
```

however, this file appears missing in the package, and is leading to a compile error
```
Module not found: Error: Can't resolve 'react-day-picker/style.css'
```

I found [this similar issue reported](https://github.com/gpbl/react-day-picker/issues/2150
) (it appears to be for version 8 docs), but there doesn't appear to be a `style.css` under the `/dist` folder either for version 9 and gives the same error.

if i change the import to the `src` directory, it does correctly compile and work, but this isn't the best solution

```
import "react-day-picker/src/style.css";
```

Curious if I'm missing something, or if this is a known issue.
Thanks in advance.

## Code 

Please include the code required to reproduce the bug.

```tsx
import { DayPicker } from "react-day-picker";
import "react-day-picker/style.css";

export function Start() {
  return (
    <DayPicker  />
  );
}
```

## Expected Behavior

able to import the "react-day-picker/style.css" and pick up the correct styling

## Actual Behavior

compile error because the style sheet appears to be missing

## Screenshots

If applicable, add screenshots or GIFs to help explain your problem.

**Comment by gpbl** [comment]
@jsmith8954 some more details about your environment? Are you using typescript, node, babel... which version? 

The export for `style.css` is defined in https://github.com/gpbl/react-day-picker/blob/114f087185d320e25f2c83e849955c2ed0175f6d/package.json#L77-L82 - so I wonder why it doesn't work for you...

**Reply by matheusmb** [reply]
Same here, not working for me.
Version: 9.0.7
Vite 5.2.11
node v20.3.1
pnpm 9.6.0

**Reply by gpbl** [reply]
@matheusmb [this project](https://stackblitz.com/edit/rdp9-vite?file=src%2FApp.tsx) is using a similar configuration with Vite and the `style.css` is imported without errors:

https://stackblitz.com/edit/rdp9-vite?file=src%2FApp.tsx

What is the difference with your config?

**Reply by sowizz** [reply]
@gpbl I've a similar issue, the file is resolved correctly by Vite, but ESLint complains that:
>Unable to resolve path to module 'react-day-picker/style.css'.
eslint([import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.29.1/docs/rules/no-unresolved.md))

Any idea how to fix it? 

My ESLint's `resolver` configuration:
```json
"import/resolver": {
  "node": {
    "paths": ["node_modules/", "src/"],
    "extensions": [
      ".js",
      ".jsx",
      ".ts",
      ".tsx",
      ".css",
      ".scss"
    ]
  }
}
```

**Comment by Frontend-io** [comment]
Having similar issue in my rollup build, however when I import the style in the global.css file, the build complains that the file is not found, however, the CSS is successfully applied when I'm testing. Is there a better way to handle this style import? 
![image](https://github.com/user-attachments/assets/fe5e046a-e515-44e6-98b2-8adde8aab898)

version: "^9.4.4"

I am importing another package style like this too and having no issues. 

![image](https://github.com/user-attachments/assets/10091b21-155b-4500-8b88-695dbb66e6c9)


I will appreciate all the help.

**Reply by gpbl** [reply]
I think it complains that the file is not found because the `postcss-import` plugin is not resolving the style.css to `/src/style.css`, as specified in package.json:

https://github.com/gpbl/react-day-picker/blob/main/package.json#L58-L69

Could you try to see if `import "react-day-picker/src/style.css"` imported without errors instead?

**Reply by adiabatic** [reply, marked as answer]
This worked for me in 9.6.7.
